apps/opik-backend/src/test/java/com/comet/opik/api/resources/v1/priv/TracesResourceTest.java

```diff
@@ -349,6 +340,34 @@ void get__whenApiKeyIsPresent__thenReturnProperResponse(String apiKey, boolean e
             }
         }
 
+        @Test
+        @DisplayName("Get traces count on previous day for all workspaces, no Auth")
+        void tracesCountForWorkspace() {
+            var workspaceName = UUID.randomUUID().toString();
+            var workspaceId = UUID.randomUUID().toString();
+
+            int tracesCnt = setupTracesForWorkspace(workspaceName, workspaceId, 1);
```

Comment:
Probably the best approach is to create at least a couple of workspace:
- N traces.
- 0 traces.
Any other case you think of.

Reply:
```suggestion
            int tracesCount = setupTracesForWorkspace(workspaceName, workspaceId, 1);
```